[pull] primary from flybywiresim:primary (#256)
Valastiri/fbw-userguide: docs/fbw-a32nx/support/index.md

## 2. Troubleshoot

!!! warning "Before reporting an issue on the FlyByWire Discord or on GitHub please follow this troubleshoot steps first."

Please follow the steps below. If you can't reproduce the issue until the last item in the list then it is most likely NOT a FlyByWire A32NX issue.

> 1. Make sure the issue is reproducible! Issues which happen only once can have many causes and can usually not be investigated or fixed as long as they can't reproduced by the FlyByWire team. To do so restart the sim and try to create the same situation again (e.g. same airports, route, livery, time of day, weather, etc.).

Reviewer: ```suggestion
1. Make sure the issue is reproducible! Issues which only occur once can have many causes and cannot usually be investigated or fixed unless they can be reproduced by the FlyByWire team. To do so restart the sim and try to create the same situation again (e.g. same airports, route, livery, time of day, weather, etc.).
```

- If the issue is reproducible do a test in a different situation (different airports, route, weather, etc.).
- If the issue is still reproducible please do a test with another aircraft (preferably a default Microsoft Flight Simulator aircraft). If the issue is reproducible with other aircraft it is most likely **not** a FlyByWire A32NX issue. If it is not reproducible with other aircraft follow the next steps.

> - To make sure the issue you are experiencing is not caused by a conflict with other add-ons or liveries remove **everything (really everything!)** your [Community Folder](#community-folder-content) and perform a [clean reinstall](../installation.md#clean-install-steps) of the A32NX with the [FlyByWire Installer](https://api.flybywiresim.com/installer){target=new}.

Reviewer 2: ```suggestion
- To make sure the issue you are experiencing is not caused by a conflict with other add-ons or liveries remove **everything (really everything!)** from your [Community Folder](#community-folder-content) and perform a [clean reinstall](../installation.md#clean-install-steps) of the A32NX with the [FlyByWire Installer](https://api.flybywiresim.com/installer){target=new}.
```

> - Also make sure no other 3rd party software like FSUIPC, SPAD.neXt, YourControls, FS2Crew, etc. is running while you are troubleshooting.

Reviewer: Minor thing. :) 

```suggestion
- Also ensure no other 3rd party software such as FSUIPC, SPAD.neXt, YourControls, FS2Crew, etc. are running while you are troubleshooting.
```

> - Read [3. Research Known Issues](#3-research-known-issues)

Reviewer 2: Small debate here - what is this was moved as the first point? I can see the issue where users might click that first and never come back to the support guide page, hence the debate. It could save them a lot of time by checking the known issues page first before proceeding down the list.

Alternatively, maybe move the #reported-issues section above the troubleshooting guide... although this encounters the same problem. 

Either way, it's technically fine as is just trying to save end-user time. Let me know your thoughts.

> - If the issue is not already known and is reproducible please do make screenshots or a video of the issue situation and continue with chapter [4. Report Issue on Discord](#4-report-issue-on-discord).

Reviewer: ```suggestion
- If the issue is not already known and is reproducible please take screenshots or a video clearly showing your issue and continue with chapter [4. Report Issue on Discord](#4-report-issue-on-discord).
```
